README.md

```diff
@@ -132,18 +132,39 @@ All classes are considered internal and are not subject to semver.
 
 ## Installation
 
-The recommended installation method is through [Composer](https://getcomposer.org/).
-Simply run
-
-    composer require felixfbecker/language-server
-
-and you will get the latest stable release and all dependencies.  
-Running `composer update` will update the server to the latest non-breaking version.
-
-After installing the language server and its dependencies,
-you must parse the stubs for standard PHP symbols and save the index for fast initialization.
-
-     composer run-script --working-dir=vendor/felixfbecker/language-server parse-stubs
+The recommended installation method is through [Composer](https://getcomposer.org/).  Follow the following installation instructions for local or global installation after installing composer on your system.
+
+### Local Installation
+Create a directory for php-language-server. Create a composer.json file in it, with the following contents:
+
+```
+{
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
I would recommend adding these lines:

https://github.com/felixfbecker/vscode-php-intellisense/blob/21d79b5fd64fb7dcc09d97febf48ade309256e65/composer.json#L7-L11

so then it's not needed to run manually.

Reply:
Okay I changed it. I am unsure at the moment if I need the following line.
```
After editing your composer.json, you can install felixfbecker/php-language-server.
composer global require felixfbecker/language-server
````

I can check if the line above is needed but it will have to be tomorrow. I'm about to turn off the computer for the day.

Reply:
I checked the lines in question above they are needed.